ci-operator/step-registry/cucushift/upgrade/eus/unpause-worker-mcp/cucushift-upgrade-eus-unpause-worker-mcp-commands.sh

```diff
@@ -53,18 +61,24 @@ function run_command() {
     eval "${CMD}"
 }
 
-
 if test -f "${SHARED_DIR}/proxy-conf.sh"
 then
     # shellcheck disable=SC1090
     source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
-unpause
-if [[ $(oc get nodes -l node.openshift.io/os_id=rhel) != "" ]]; then
-    echo "Found rhel worker, this step is supposed to be used in eus upgrade, skipping mcp checking here, need to check it after rhel worker upgraded..."
-    run_command "oc get machineconfigpools"
-    run_command "oc get node -owide"
-else
-    check_mcp
-fi
+IFS=" " read -r -a arr <<<"$PAUSED_MCP_NAME"
+for mcp in "${arr[@]}";
+do
+    check_mcp $mcp "False"
```

Comment:
As we discussed, we also need check un_paused_mcp upgraded before unpause, right? And it is consistent with current implementation in v4-installer-auto repo.